AirCasting/SearchAndFollow/SearchMap/BottomCard.swift

```diff
@@ -45,7 +45,7 @@ struct BottomCardView: View {
         .background(
             Group {
                 Color.white
-                    .shadow(color: .sessionCardShadow, radius: 9, x: 0, y: 1)
+                    .shadow(color: .bottomCardShadow, radius: 9, x: 0, y: 1)
```

Comment:
Here you u`re using the old-style version approach, please change to the new one 😄

Reply:
Yeah, looks good I think. Please change this if possible